src/main/kotlin/models/UserModel.kt

```diff
@@ -1,21 +1,20 @@
-package repo
+package models
 
 import kotlinx.serialization.Serializable
 import org.json.JSONObject
 
 @Serializable
```

Comment:
you can remove @Serializable annotation now as explicit serialization not required in case of mongo db